backend/app/services/adapters/team_kinds.py

```
@@ -1996,5 +1996,330 @@ def get_team_input_parameters(
+            The newly created Bot Kind object
+        """
+        bot_crd = Bot.model_validate(bot.json)
+
+        # Check if bot with same name already exists in target namespace
+        existing_bot = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == "Bot",
+                Kind.name == bot.name,
+                Kind.namespace == target_namespace,
+                Kind.is_active == True,
```

Ruff (0.14.11)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)